.github/workflows/ci.yml

```diff
@@ -429,6 +429,7 @@ jobs:
             os: windows-2019-8core-32gb
     timeout-minutes: 600
     runs-on: "${{ matrix.os }}"
+    continue-on-error: true
```

Comment:
You seem to have a bunch of accidental diffs in here

Reply:
They're not accidental, they are for testing building all targets with my patches. I'll remove them when upstream releases the changes I sent.